app/models/manageiq/providers/inventory/persister/builder/persister_helper.rb

```diff
@@ -135,10 +131,9 @@ def name_references(collection)
   private
 
   def add_collection_for_manager(manager_type, collection_name, extra_properties = {}, settings = {}, &block)
-    parent_manager = send("#{manager_type}_manager")
-    settings.reverse_merge!(:shared_properties => {:parent => parent_manager})
```

Comment:
This was a bit of hoop jumping as a result of shared_properties not intending to be changed per-collection